src/simplewallet/simplewallet.cpp

```diff
@@ -5175,7 +5178,7 @@ bool simple_wallet::donate(const std::vector<std::string> &args_)
   local_args.push_back(amount_str);
   if (!payment_id_str.empty())
     local_args.push_back(payment_id_str);
-  message_writer() << tr("Donating ") << amount_str << " to The Monero Project (donate.getmonero.org or "<< MONERO_DONATION_ADDR <<").";
+  message_writer() << tr("Donating ") << amount_str << " WOW to help get some stripers through college.";
```

Comment:
Maybe a bit pedantic, but the unit may be changed with the `set unit <unit_name>` command, in which case the shown amount wouldn't be in WOW. The name of the currently selected unit can be obtained by `cryptonote::get_unit(cryptonote::get_default_decimal_point())`. But first of all, in order for this to make sense, the `get_unit()` function in cryptonote_format_util.cpp needs to be updated.